src/components/MainTable/MainTable.js

```diff
@@ -252,4 +263,9 @@ MainTable.propTypes = {
   sortFunction: PropTypes.func
 };
 
+// Specifies the default values for props:
+MainTable.defaultProps = {
+  emptyStateMsg: "No data to display"
+};
```

Comment:
There's no need to use defaultProps with ES6 default values, better to set this on l151.